drivers/hmis/app/graphql/mutations/delete_service.rb

```diff
@@ -5,8 +5,20 @@ class DeleteService < BaseMutation
     field :service, Types::HmisSchema::Service, null: true
 
     def resolve(id:)
-      record = Hmis::Hud::Service.viewable_by(current_user).find_by(id: id)
-      default_delete_record(record: record, field_name: :service, permissions: :can_edit_enrollments)
+      # ID should be a HmisService view ID, which starts with 1 or 2.
+      return { errors: [HmisErrors::Error.new(:service, :not_found)] } if id.length < 2 || !['1', '2'].include?(id.first)
```

Comment:
Can we change these to something like:
```suggestion
      return { errors: [HmisErrors::Error.new(:service, :not_found)] } if id.length < 2 || Hmis::Hud::HmisService.valid_id?(id)
```
That way, eventually, if we add other service types, we don't have to go around looking for `!['1', '2'].include?(id.first)`

Reply:
Yep